packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -1,123 +1,126 @@
+  maxZoom: 10,
+  minZoom: 0.1,
+  zoomInValue: 1.2,
+  zoomOutValue: 0.8
+}
+
 function SingleImageViewer({
-  children,
-  enableInteractionLayer = false,
+  enableInteractionLayer = true,
+  enableRotation = DEFAULT_HANDLER,
   frame = 0,
-  height,
-  limitSubjectHeight = false,
-  onKeyDown = () => true,
-  rotate = 0,
-  scale = 1,
-  svgMaxHeight = null,
+  imgRef,
+  invert = false,
+  move = false,
+  naturalHeight,
+  naturalWidth,
+  onKeyDown = DEFAULT_HANDLER,
+  panning = true,
+  rotation = 0,
+  setOnPan = DEFAULT_HANDLER,
+  setOnZoom = DEFAULT_HANDLER,
+  src,
   subject,
-  title = {},
-  viewBox,
-  width,
+  title = undefined,
   zoomControlFn = null,
-  zooming = false
+  zooming = true
 }) {
-  const canvasLayer = useRef()
-  const canvas = canvasLayer.current
-  const transform = `rotate(${rotate} ${width / 2} ${height / 2})`
+  useEffect(function onMount() {
+    enableRotation()
+  }, [])
+
+  const singleImageCanvasProps = {
+    enableInteractionLayer,
+    frame,
+    imgRef,
+    invert,
+    move,
+    naturalHeight,
+    naturalWidth,
+    onKeyDown,
+    rotation,
+    src,
+    subject
+  }
 
   return (
-    <SVGContext.Provider value={{ canvas, viewBox, rotate, width, height }}>
+    <>
       {zoomControlFn && (
         <ZoomControlButton
           onClick={zoomControlFn}
           zooming={zooming}
         />
       )}
       <Box
+        align='flex-end'
         animation='fadeIn'
         overflow='hidden'
         width='100%'
-        align='flex-end'
       >
-        <PlaceholderSVG
-          focusable
-          maxHeight={svgMaxHeight}
-          maxWidth={limitSubjectHeight ? `${width}px` : '100%'}
-          onKeyDown={onKeyDown}
-          tabIndex={0}
-          viewBox={`0 0 ${width} ${height}`}
-          xmlns='http://www.w3.org/2000/svg'
-        >
-          {title?.id && title?.text && (
-            <title id={title.id}>{title.text}</title>
-          )}
-          <g
-            transform={transform}
-          >
-            <SVGImageCanvas
-              ref={canvasLayer}
-              viewBox={viewBox}
-            >
-              {children}
-              {enableInteractionLayer && (
-                <InteractionLayer
-                  frame={frame}
-                  height={height}
-                  scale={scale}
-                  subject={subject}
-                  width={width}
-                />
-              )}
-            </SVGImageCanvas>
-          </g>
-        </PlaceholderSVG>
+        {title?.id && title?.text && (
+          <title id={title.id}>{title.text}</title>
+        )}
+          <VisXZoom
+            height={naturalHeight}
+            panning={panning}
+            setOnPan={setOnPan}
+            setOnZoom={setOnZoom}
+            width={naturalWidth}
+            zoomConfiguration={DEFAULT_ZOOM_CONFIG}
+            zoomingComponent={SingleImageCanvas}
+            zooming={zooming}
+            {...singleImageCanvasProps}
+          />
```

Comment:
See https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ZoomingScatterPlot/ZoomingScatterPlot.js#L206-L224 for existing implementation of VisXZoom. `constrain` could be added to the SingleImageViewer potentially.